ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -132,48 +132,48 @@ class TooltipView: UIView {
     private func updateArrowDirectionAndTooltipRect(for message: String, title: String? = nil, tokenSet: TooltipTokenSet) {
         let preferredBoundingRect = boundingRect.inset(by: anchorViewInset(for: preferredArrowDirection))
         let backupBoundingRect = boundingRect.inset(by: anchorViewInset(for: preferredArrowDirection.opposite))
-        guard let window = anchorView.window else {
-            preconditionFailure("Can't find anchorView's window")
-        }
-        let isAccessibilityContentSize = window.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
-        let preferredSize = TooltipView.sizeThatFits(preferredBoundingRect.size,
-                                                     message: message,
-                                                     title: title,
-                                                     isAccessibilityContentSize: isAccessibilityContentSize,
-                                                     arrowDirection: preferredArrowDirection,
-                                                     tokenSet: tokenSet)
-        let backupSize = TooltipView.sizeThatFits(backupBoundingRect.size,
-                                                  message: message,
-                                                  title: title,
-                                                  isAccessibilityContentSize: isAccessibilityContentSize,
-                                                  arrowDirection: preferredArrowDirection.opposite,
-                                                  tokenSet: tokenSet)
-        var usePreferred = true
-        if (preferredArrowDirection.isVertical &&
-            preferredBoundingRect.height < preferredSize.height &&
-            backupBoundingRect.height >= backupSize.height) ||
-            (!preferredArrowDirection.isVertical &&
-             preferredBoundingRect.width < preferredSize.width &&
-             backupBoundingRect.width >= backupSize.width) {
-            usePreferred = false
-        }
+        if let window = anchorView?.window {
```

Comment:
Per the Microsoft Swift guide, these large `if let x = z?.x` statements should instead be structured as a `guard let`, which will also remove the need to change the indentation of the below block.